src/officehours_api/views.py

```diff
@@ -290,3 +295,50 @@ def get_queryset(self):
 class AttendeeDetail(DecoupledContextMixin, generics.RetrieveAPIView):
     queryset = Attendee.objects.all()
     serializer_class = AttendeeSerializer
+
+class ExportMeetingStartLogs(APIView):
+    permission_classes = [IsAuthenticated]
+
+    def get(self, request, queue_id=None, format=None):
+        username = request.user.username
+
+        # Query from the Queue hosts table to find all the queues the user is a host of, include deleted
+        queues_user_is_in = Queue.all_objects.filter(hosts__in=[request.user]).values_list('id', flat=True)
+
+
+        # If they specify a single queue id, check if they're actually in it and return that
+        if queue_id:
+            # Security check that they are actually in this queue
+            if queue_id not in queues_user_is_in:
+                return Response({'detail': 'You are not a host of this queue.'}, status=status.HTTP_403_FORBIDDEN)
+            else:
+                queues_user_is_in = [queue_id]
+                filename = f"meeting_start_logs_{username}_queue_{queue_id}.csv"
+        # Otherwise, get all the logs for the queues the user is a host of
+        else:
+            filename = f"meeting_start_logs_{username}.csv"
+        
+        logger.info(f"User {username} requested to export meeting start logs for queues {queues_user_is_in}.")
+        response = HttpResponse(content_type='text/csv')
+        response['Content-Disposition'] = f'attachment; filename="{filename}"'
+
+        writer = csv.writer(response)
+        with connection.cursor() as cursor:
+            # Generate a string of placeholders, one for each item in the list
+            queue_id_placeholders = ', '.join(['%s'] * len(queues_user_is_in))
+
+            # Add a query to just get the logs for the queues_user_is_in
+            cursor.execute(f"SELECT * FROM meeting_start_logs where queue_id in ({queue_id_placeholders})", queues_user_is_in)
```

Comment:
![image](https://github.com/tl-its-umich-edu/remote-office-hours-queue/assets/147423601/4581f455-4c44-42b6-aa32-cd7348421b3c)

if the user has no queues to export, we get a 500 error (where we probably want [200/204?](https://stackoverflow.com/questions/11746894/what-is-the-proper-rest-response-code-for-a-valid-request-but-empty-data)). alternatively an empty csv would work as well